src/main.py

```
@@ -2174,6 +2234,73 @@ async def _stop_mcp_lifespan():
+            raise
+
+        # One-shot JSON->DB migration + test-fixture cleanup. Runs once
+        # per install, idempotent via migration_status rows.
+        try:
+            from db.engine import SessionLocal as _SL
+            from db.migrations_runtime import run as run_runtime_migration
+            if _SL is not None:
+                async with _SL() as _session:
+                    await run_runtime_migration(_session)
+                    await _session.commit()
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)